test/fuse/linux/fuse_base.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <sys/uio.h>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace gvisor {
+namespace testing {
+
+constexpr char MOUNT_POINT[] = "/mnt";
```

Comment:
Recommendation:

Thinking maybe we can generate a random string with time stamp to create a directory in `/tmp` as mount point.

With that, no need to worry about if the machine already mounted anything on `/mnt`, and multiple fuse_tests can be run concurrently.

Reply:
It's totally fine since each test case runs in an isolated runsc runtime. They can be tested concurrently and safely. See `test/runner/runner.go` for further details.